backend/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
should we have a scripts folder? or who cares?

Reply:
Not a bad idea. The moment we have two scripts or I have to add an extra config file or something, I'll put them in a folder to keep things tidy.